boa/src/context.rs

```diff
@@ -253,11 +264,13 @@ impl Default for Context {
             executor,
             #[cfg(feature = "console")]
             console: Console::default(),
+            typed_array_prototype: Default::default(),
```

Comment:
Shouldn't this be done in https://github.com/boa-dev/boa/blob/master/boa/src/builtins/mod.rs#L28-L52 ?

Reply:
I was thinking that too actually. The only reason it's there is because the constructor for `TypedArray` needs `&mut Context` because I was thinking it might be possible to put all of the array methods in there.

Reply:
Yep, the thing here is that all typed arrays share base prototype, but that prototype is not available anywhere in the global object (only getting the prototype of a typed array).